README.md

```diff
@@ -1,64 +1,83 @@
-# **OSSF Project/WG Name**
+# OpenSSF Malicious Packages
 
-[Brief description of the initiative]
+This repository is a collection of reports of malicious packages identified in
+Open Source package repositories consumable via the
+[Open Source Vulnerability (OSV)](https://osv.dev) format.
 
+This project is closely related to the [OpenSSF
+Package Analysis project](https://github.com/ossf/package-analysis).
 
-## 
-**Motivation**
+## About
 
-[Background / use cases of the problem to be solved]
+### Background
 
+Attacks against open source ecosystems are gaining popularaity. Typosquating,
+dependency confusion, account takeovers, etc are happening more frequently each
+year.
 
-## 
-**Objective**
+While some protection can be found through various security solutions, there
+does not exist any comprehensive database of malicious packages published to
+open source package repositories.
 
-[What is to be achieved with this initiative]
+### Objective
 
-[OKRs - OPTIONAL]
+The aim of this project and repository is to be a comprehensive, high quality,
+open source database of reports of malicious packages published on open source
+package repositories.
 
+These public reports help protect the open source community, and provide a data
+source for the security community to improve their ability to find and detect
+new open source malware.
 
-## 
-**Scope**
+### Scope
 
-[What is in and out of scope]
+What is in scope?
 
+- any package that belongs to an ecosystem supported by the [OSV Schema](https://ossf.github.io/osv-schema/)
+- malicious packages published under typosquating or dependency confusion type attacks
+- malicious packages published through account takeover
+- security researcher activity
 
-## 
-**Prior Work**
+Out-of-scope:
 
+- vulnerability reports
+- compromised infrastructure
+- non-malicious packages
 
+### Prior Work
 
-*   List of prior and/or related projects
+- GitHub's [Advisory Database (filtered by malware)](https://github.com/advisories?query=type%3Amalware), for the NPM ecosystem.
 
-# 
-**Get Involved**
+## Get Involved
 
-*   Official communications occur on the [ADD LINK TO YOUR WG MAILING LIST] (ex: https://lists.openssf.org/g/openssf-tac/topics).  \
-[Manage your subscriptions to Open SSF mailing lists](https://lists.openssf.org/g/main/subgroups).
-*   [Add Slack information if availabable]
+### Contribute Malicious Package Reports
+
+#### Pull Requests
+
+We accept new reports, and updates to existing reports.
 
-### 
-Quick Start
+#### Automated Sources (Coming Soon!)
 
-*   Areas that need contributions
-*   Build information if applicable
-*   Where to file issues
-*   Etc.
+If you regularly produce high-quality, low false positive, detections and
```

Comment:
suggest "high-quality detections with few false positives, and have them accumulating..."